### ISSUE TITLE
Can't gain xp with mod enabled on dedicated server

### ISSUE THREAD

**Issue by suxel33** [ineedbots/iw3_bot_warfare#36]
Hello I succesfully set up this mod on cod4x with a dedicated server that shows up on the master list, everything works perfectly except that I can't gain xp, but the bots do? is there any solution to this?

**Comment by hackmoore** [comment]
Also having this issue. Further to this, any xp and loadouts that were enabled on the account are no longer available, it seems to reset the rank back to 0.

**Comment by fonix4000** [comment]
Issue is not related to the bot_warfare mod but with your server

**Comment by hackmoore** [comment]
How do you know? If I remove bot_warfare, the problem is resolved.

**Comment by ForeskinJohn** [comment]
has anyone fixed this issue? what is going on here?

**Comment by ForeskinJohn** [comment]
and to reply to fonix4000 a lot of us are using the default configuration that comes with the mod, so either way if its not an issue with the mod itself, something is changed, and i cant for the life of me find out what, i remember having this issue before but now there seems to be no way to resolve it.

**Comment by fonix4000** [comment]
look for level.rankedMatch in globallogic or wherever you have put it

**Comment by ForeskinJohn** [comment]
yeah i got my rank working, but now the ai wont use killstreaks, and my game just decided to crash, i give up for now, gonna wait until 2.3 comes out and see if this is fixed.

**Comment by ForeskinJohn** [comment]
to add, make sure to install the server and client from the cod4x releases page on the forum, and do not update it. thats how i got rank working, i assume there are some dll files missing on the github page, as all it is is the cod deadrun and plugins folder.

**Comment by ineedbots** [comment]
Bot Warfare doesn't do anything about ranking, that would be the vanilla's scripts setting level.rankedMatch and dedicated server.
